Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM praekeltfoundation/django-bootstrap:py3.6
+
+COPY ./requirements.txt /app/
+COPY ./setup.py /app/
+
+RUN pip install -r /app/requirements.txt
+
+COPY . /app
+
+# temporary untill there is a new PyCap Release
+ENV DJANGO_SETTINGS_MODULE "config.settings.production"
+RUN SECRET_KEY=placeholder ALLOWED_HOSTS=placeholder python manage.py collectstatic --noinput
+CMD ["config.wsgi:application"]
```

Comment:
This isn't 100% necessary, but there are some performance changes you can make to the gunicorn config, https://github.com/praekeltfoundation/flow-results/blob/main/Dockerfile#L10-L16 is a good start.